### PR TITLE
[export-file-stix] fix export list (#1387)

### PR DESCRIPTION
**Pull request by SouadHadjiat**
### Proposed changes

**Related PR in client-python :** https://github.com/OpenCTI-Platform/client-python/pull/448

* Fix export list (https://github.com/OpenCTI-Platform/connectors/issues/1387)
* Fix export relationships (https://github.com/OpenCTI-Platform/opencti/issues/4253)

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/1387

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Review by SarahBocognano — COMMENTED** [review]
I have an error when trying to export a full export : in generate_export\n    original_entity_type = entity[\"entity_type\"]\nTypeError: string indices must be integers"}

**Review by SarahBocognano — APPROVED** [review]

File: internal-export-file/export-file-stix/src/export-file-stix.py [hunk 1]
```diff
@@ -52,11 +52,6 @@ def _process_message(self, data):
             )
 
         else:  # export_scope = 'query' or 'selection'
-            if "element_id" in data:
```
Comment (RomuDeuxfois) [inline internal-export-file/export-file-stix/src/export-file-stix.py:55]: I think we need to keep this to be able to export only selected entities

**Comment by RomuDeuxfois** [comment]
Bug foud :
1. Go to sighting
2. Filter on sighted in/at
3. Export

Results : bundle is not filtered

**Comment by RomuDeuxfois** [comment]
Bug foud :
1. Go to a stix file
2. Go to data tab
3. Export

Results : bundle is empty

File: internal-export-file/export-file-stix/src/export-file-stix.py (hunk 1 above)
Comment (RomuDeuxfois) [inline internal-export-file/export-file-stix/src/export-file-stix.py:55]: This property was not used ✅

**Review by RomuDeuxfois — APPROVED** [review]

**Pull request merged**